### PR TITLE
fix(storefront): BCTHEME-1646 Top Global Region Image Widget overlaps the mobile menu

### PR DESCRIPTION
**Pull request by bc-yevhenii-buliuk**
#### What?

This PR fixes a bug with overlaps in the mobile menu.

#### Tickets / Documentation

- [BCTHEME-1646](https://bigcommercecloud.atlassian.net/browse/BCTHEME-1646)

#### Screenshots / Video

**Bug**:

https://github.com/bigcommerce/cornerstone/assets/82589781/e0060b61-41c5-4174-8c57-86a6fe13246a

**Fix**:

https://github.com/bigcommerce/cornerstone/assets/82589781/6584bad7-9f41-46d9-a934-a5e836ee0cb3

[BCTHEME-1646]: https://bigcommercecloud.atlassian.net/browse/BCTHEME-1646?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by bc-yevhenii-buliuk** [comment]
@bigcommerce/team-themes how do you think, is it Ok that the yellow banner overlaps the main carousel as well?

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Display fees on cart page [#2376](https://github.com/bigcommerce/cornerstone/pull/2376)
 - Replace Twitter logo with X logo within social sharing and social link components [#2387](https://github.com/bigcommerce/cornerstone/pull/2387)
 - Added nvm config [#2389](https://github.com/bigcommerce/cornerstone/pull/2389)
-- Displaying the Hidden cart_order_source Input Field on PDP page [#2392](https://github.com/bigcommerce/cornerstone/pull/2392)
```
Comment (bc-yevhenii-buliuk) [inline CHANGELOG.md:11]: there was unnecessary extra space

**Review by BC-krasnoshapka — APPROVED** [review]

**Review by bc-vlad-dlogush — APPROVED** [review]

**Review by yurytut1993 — APPROVED** [review]
lgtm

**Review by bc-alexsaiannyi — APPROVED** [review]
🔥

**Pull request merged**